ci/pipeline/ray_coverage.py

```diff
@@ -1,11 +1,11 @@
 import boto3
 import click
-import logging
 import os
 import subprocess
 import sys
 import tempfile
 from datetime import date
+from ray_logger import get_logger
```

Comment:
if we are moving the logger out.. maybe it is time to build python files in `ci/pipeline` in bazel now..

Reply:
right, they have a tendency to become a proper library at some point

Reply:
Discuss offline, not gonna do it at this point